bin/run_tests

```diff
@@ -28,4 +28,4 @@ run_docker_test seth_blockhash_test
 run_docker_test seth_blocknum_test
 run_docker_test seth_timestamp_test
 run_docker_test seth_perm_test
-run_docker_test ./rpc/comp_seth_rpc.yaml
+# run_docker_test ./rpc/comp_seth_rpc.yaml
```

Comment:
When will this be re-enabled?  Perhaps leave a comment above this line to indicated why this commented code should be left here.